web/src/main/angular/src/globals.d.ts

```diff
@@ -111,27 +120,32 @@ interface ILinkInfo {
     fromAgent?: string[];
     hasAlert: boolean;
     histogram: IResponseTime | IResponseMilliSecondTime;
+    responseStatistics: IResponseStatistics;
     key: string;
     slowCount: number;
     sourceHistogram?: { [key: string]: IResponseTime | IResponseMilliSecondTime };
+    sourceResponseStatistics?: { [key: string]: IResponseStatistics };
     sourceInfo: ISourceInfo;
     sourceTimeSeriesHistogram?: { [key: string]: IHistogram }[];
     targetHistogram?: { [key: string]: IResponseTime | IResponseMilliSecondTime };
+    targetResponseStatistics?: { [key: string]: IResponseStatistics };
     targetInfo: ISourceInfo;
     timeSeriesHistogram: IHistogram[];
     to: string;
     toAgent?: string[];
     totalCount: number;
 }
 interface INodeInfo {
-    agentHistogram?: { [key:string]: IResponseTime | IResponseMilliSecondTime }[];
-    agentTimeSeriesHistogram?: { [key:string]: IHistogram[] };
+    agentHistogram?: { [key: string]: IResponseTime | IResponseMilliSecondTime }[];
+    agentTimeSeriesHistogram?: { [key: string]: IHistogram[] };
     agentIds: string[];
     applicationName: string;
     category: string;
     errorCount: number;
     hasAlert: boolean;
     histogram: IResponseTime | IResponseMilliSecondTime;
+    responseStatistics: IResponseStatistics;
+    agentResponseStatistics: { [key: string]: IResponseStatistics};
```

Comment:
Declare it as an optional property.
```suggestion
    agentResponseStatistics?: { [key: string]: IResponseStatistics};
```